src/services/AdvancedTradingView/advancedTradingView.types.ts

```diff
@@ -75,8 +75,11 @@ export type AdapterFetchDetails = {
   amountToFetch: number
   isFirstFetch: boolean
   abortController: (id: string) => AbortSignal
+  refreshing?: boolean
```

Comment:
Let's use the same convention as `isFirstFetch` in this type

Reply:
These are different; `isFirstFetch` was there to determine first fetch from the others and it wasn't implemented properly as the state was reset all the time. Property I created - `refreshing`, differentiates getting the latest data from the one you get while scrolling down.

Reply:
Got it